util/hookable/hookable.cpp

```diff
@@ -342,6 +404,115 @@ J9HookRegister(struct J9HookInterface **hookInterface, uintptr_t taggedEventNum,
 					(emptyRecord->next->agentID >= agentID)))
 		) {
 			emptyRecord->function = function;
+			emptyRecord->callsite = NULL;
```

Comment:
J9HookRegister were used in omr, j9vm and jit, in order to smooth migration from J9HookRegister() to J9HookRegisterWithCallSite(), we temporarily keep old interface J9HookRegister to avoid dependence issue between omr and other components.